specs/waku/waku-rpc-api.md

```diff
@@ -0,0 +1,410 @@
+---
+title: Waku RPC API
+version: 1.0.0
+status: Draft
+authors: Dean Eigenmann <dean@status.im>
+---
+
+## Table of Contents
+
+1. [Abstract](#abstract)
+2. [Wire Protocol](#wire-protocol)
+    1. [Transport](#transport)
+    1. [Objects](#objects)
+    1. [Methods](#methods)
+3. [Copyright](#copyright)
+
+## Abstract
+
+In this specification we describe the RPC API that Waku nodes SHOULD adhere to. The unified API allows clients to easily
```

Comment:
Might be worth mentioning somewhere that this is a privileged API and not like some REST API, since you store keys for filter

Reply:
Yes, I would also change SHOULD to MAY. As it is really something for a client to decide on imo.
We could even add some sentence that this typically would be restricted to access from localhost default.